### PR TITLE
Added Lease filters and anomalies to advanced property | psp-4845 & psp-6230

### PR DESCRIPTION
**Pull request by FuriousLlama**
image

### CODE REVIEW

File: source/backend/dal/Repositories/PropertyRepository.cs [hunk 1]
```diff
@@ -381,6 +381,7 @@ public HashSet<long> GetMatchingIds(PropertyFilterCriteria filter)
         {
             var query = Context.PimsProperties.AsNoTracking();
```
Comment (devinleighsmith) [inline source/backend/dal/Repositories/PropertyRepository.cs:382]: @FuriousLlama I feel like I'm missing something obvious, how could this possibly be working if there are no includes on the PimsProperties query?

**Comment by devinleighsmith** [comment]
minor display issue, could fix this as a bug due to time constraints: 
image

**Comment by devinleighsmith** [comment]
Huh, we probably should have implemented a refresh button or clear button for the whole filter but I don't see that in the requirements or mockup so should be fine for this initial release.

**Comment by devinleighsmith** [comment]
well, despite the missing includes everything works, so will approve. I must just be missing something.

**Review by devinleighsmith — APPROVED** [review]

File: source/backend/dal/Repositories/PropertyRepository.cs (hunk 1 above)
Comment (FuriousLlama) [reply source/backend/dal/Repositories/PropertyRepository.cs:382]: Afaik this means, loading all properties. The individual blocks below make sure to add conditionals

**Pull request merged**